### PR TITLE
[KOGITO-2551] - Review docs links and wording for CSV and CLI command

### PR DESCRIPTION
**Pull request by ricardozanini**
See: https://issues.redhat.com/browse/KOGITO-2551

@sterobin mind taking a look?

Signed-off-by: Ricardo Zanini <zanini@redhat.com>

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ spec:
   subresources: {}
   validation:
     openAPIV3Schema:
-      description: KogitoApp is a project prescription running a Kogito Runtime Service.
+      description: KogitoApp is a project prescription running a Kogito service.
```
Comment (sterobin) [inline deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml:41]: @ricardozanini Some descriptions end with a period, some don't. They all should, for consistency, but honestly up to you how particular you want to be, given its purpose.

Comment (ricardozanini) [reply deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml:41]: Updated!

File: deploy/crds/app.kiegroup.org_kogitojobsservices_crd.yaml [hunk 2]
```diff
@@ -256,13 +256,13 @@ spec:
               type: integer
             replicas:
               description: 'Number of replicas that the service will have deployed
```
Comment (Sgitario) [inline deploy/crds/app.kiegroup.org_kogitojobsservices_crd.yaml:258]: Why this description contains ' ?

File: deploy/crds/app.kiegroup.org_kogitomgmtconsoles_crd.yaml [hunk 3]
```diff
@@ -171,13 +171,13 @@ spec:
               type: object
             replicas:
               description: 'Number of replicas that the service will have deployed
```
Comment (Sgitario) [inline deploy/crds/app.kiegroup.org_kogitomgmtconsoles_crd.yaml:173]: Same as above: why contains ' characters?

File: deploy/crds/app.kiegroup.org_kogitoruntimes_crd.yaml [hunk 4]
```diff
@@ -244,18 +244,18 @@ spec:
                     On OpenShift, OLM should install it for you. If running on Kubernetes
                     without OLM installed, please install Strimzi Operator first.
                     Set this to false and fill other properties to provide your own
-                    infrastructure
+                    infrastructure.
                   type: boolean
               type: object
             replicas:
               description: 'Number of replicas that the service will have deployed
```
Comment (Sgitario) [inline deploy/crds/app.kiegroup.org_kogitoruntimes_crd.yaml:251]: Same as above: why contains ' characters?

File: deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitoapps_crd.yaml [hunk 5]
```diff
@@ -53,22 +53,22 @@ spec:
         metadata:
           type: object
         spec:
-          description: KogitoAppSpec defines the desired state of KogitoApp
+          description: KogitoAppSpec defines the desired state of KogitoApp.
           properties:
             build:
-              description: 'S2I Build configuration Default value: nil'
+              description: 'S2I Build configuration. Default value: nil'
```
Comment (Sgitario) [inline deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitoapps_crd.yaml:59]: Same as above: why contains ' characters?

File: deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitoapps_crd.yaml [hunk 6]
```diff
@@ -413,13 +413,13 @@ spec:
               type: object
             replicas:
               description: 'Number of replicas that the service will have deployed
```
Comment (Sgitario) [inline deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitoapps_crd.yaml:415]: Same as above: why contains ' characters?

File: deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitoapps_crd.yaml [hunk 7]
```diff
@@ -437,68 +437,68 @@ spec:
...
-                Default value: quarkus'
+              description: 'The name of the runtime used, either Quarkus or SpringBoot.
+                Default value: quarkus.'
               enum:
               - quarkus
               - springboot
               type: string
             service:
-              description: 'Kubernetes Service configuration Default value: nil'
+              description: 'Kubernetes Service configuration. Default value: nil'
```
Comment (Sgitario) [inline deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitoapps_crd.yaml:447]: Same as above: why contains ' characters?

File: deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitobuilds_crd.yaml [hunk 8]
```diff
@@ -244,16 +244,16 @@ spec:
                   type: object
               type: object
             runtime:
-              description: 'Which runtime Kogito Runtime base image to use when building
-                the Kogito Service. If "BuildImage" is set, this value is ignored
-                by the operator. Default value: quarkus'
+              description: 'Which runtime Kogito service base image to use when building
+                the Kogito service. If "BuildImage" is set, this value is ignored
+                by the operator. Default value: quarkus.'
```
Comment (Sgitario) [inline deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitobuilds_crd.yaml:249]: Same as above: why contains ' characters?

File: deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitobuilds_crd.yaml [hunk 9]
```diff
@@ -244,16 +244,16 @@ spec:
...
               enum:
               - quarkus
               - springboot
               type: string
             runtimeImage:
-              description: 'Image used as the base image for the final Kogito Runtime
-                Service. This image only has the required packages to run the application.
+              description: 'Image used as the base image for the final Kogito service.
```
Comment (Sgitario) [inline deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitobuilds_crd.yaml:255]: Same as above: why contains ' characters?

File: deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitobuilds_crd.yaml [hunk 10]
```diff
@@ -285,27 +285,27 @@ spec:
             type:
               description: 'Sets the type of build that this instance will handle:
                 Binary - takes an uploaded binary file already compiled and creates
-                a Kogito Runtime Service image from it RemoteSource - pulls the source
-                code from a Git repository, builds the binary and then the final Kogito
-                Runtime Service image LocalSource - takes an uploaded resource file
-                such as DRL (rules), DMN (decision) or BPMN (process), builds the
-                binary and the final Kogito Runtime Service image'
+                a Kogito service image from it. RemoteSource - pulls the source code
```
Comment (Sgitario) [inline deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitobuilds_crd.yaml:288]: Same as above: why contains ' characters?

File: deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitodataindices_crd.yaml [hunk 11]
```diff
@@ -239,18 +239,18 @@ spec:
...
                     without OLM installed, please install Strimzi Operator first.
                     Set this to false and fill other properties to provide your own
-                    infrastructure
+                    infrastructure.
                   type: boolean
               type: object
             replicas:
               description: 'Number of replicas that the service will have deployed
-                in the cluster Default value: 1'
+                in the cluster. Default value: 1.'
```
Comment (Sgitario) [inline deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitodataindices_crd.yaml:247]: Same as above: why contains ' characters?

File: deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitojobsservices_crd.yaml [hunk 12]
```diff
@@ -256,13 +256,13 @@ spec:
               type: integer
             replicas:
               description: 'Number of replicas that the service will have deployed
-                in the cluster Default value: 1'
+                in the cluster. Default value: 1.'
```
Comment (Sgitario) [inline deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitojobsservices_crd.yaml:259]: Same as above: why contains ' characters?

File: deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitomgmtconsoles_crd.yaml [hunk 13]
```diff
@@ -171,13 +171,13 @@ spec:
               type: object
             replicas:
               description: 'Number of replicas that the service will have deployed
-                in the cluster Default value: 1'
+                in the cluster. Default value: 1.'
```
Comment (Sgitario) [inline deploy/olm-catalog/kogito-operator/0.12.0/app.kiegroup.org_kogitomgmtconsoles_crd.yaml:174]: Same as above: why contains ' characters?

**Comment by ricardozanini** [comment]
> lgtm. Just minor comments about some descriptions contains ' characters. If there is a reason to have them, feel free to ignore my comments.

Those are generated files, we can't control whether or not those quotes are added. I believe the tool add them when there's a limit of characters and the sentence needs to break a line.

**Comment by spolti** [comment]
just a conflict that needs to be fixed :)

**Pull request merged**